assets/css/style.css

```diff
@@ -1,3 +1,6 @@
 .hide {
   display: none;
 }
+.done{
```

Comment:
separate style rules by line breaks for readability

```suggestion
}

.done{
```